examples/riot/src/blinky/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of your application
+APPLICATION = lf-test
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../../../../RIOT
```

Comment:
I have specified the RIOT folder to be one higher thant the root of this project. So it is easy to just clone and put the RIOT folder next to this project.

Maybe we can improve the ergonomix here by including RIOT as a submodule?
If this is the way to go, I was not yet sure where to put it. Should we put the RIOT submodule just in the examples/riot folder?

Reply:
Yes including this project into riot as a package is easy and a common way to do it.
But for testing this project on riot and doing active development on it I find the RIOT package system inkonvenient. It requires you to point to a specific commit ID and we always need to update the RIOT package to new versions.

Thats why I added this example within this project. So I see 2 options: 
- Option 1: Add RIOT as a submodule to the examples/riot folder, which does not mean this project depends on RIOT, but just makes it more easy for new people to run the riot examples in here.
- Option 2: Keep as is and just explain in the readme that one needs to clone RIOT next to this project on their file system to run the examples.

Both ways are totally reasonable for me, but feel free to come up with an even better solution :);)

Reply:
I prefer 2 but I would like to not rely on relative paths, what about requiring an environmental variable, e.g. `RIOT_PATH` or `REACTOR_UC_RIOT_PATH` or something?

Reply:
In the marked line you can see that there already is an environment varialble: "RIOTBASE" this is the common variable name we use everywhere in RIOT OS applications. The relative path is just a fallback if that variable is not specified